requirements.txt

```diff
@@ -2,3 +2,4 @@ beautifulsoup4==4.4.1
 python-magic==0.4.10
 requests==2.8.1
 wheel==0.24.0
```

Comment:
The `wheel` and the `docopt` spec is not required.

And the `requests` module got upgraded to `2.10.*`. I am not sure on that. So check it out.